Fabric.Terminology.API/appsettings.json

```diff
@@ -1,19 +1,19 @@
 ﻿{
-  "BaseTerminologyEndpoint": "https://hc2259.hqcatalyst.local/terminology",
-  "SwaggerRootBasePath":  "", 
+  "BaseTerminologyEndpoint": "**BaseTerminologyEndpoint**",
+  "SwaggerRootBasePath": "**SwaggerRootBasePath**",
   "TerminologySqlSettings": {
-    "ConnectionString": "Data Source=HCS-GM0001;Integrated Security=SSPI;Initial Catalog=Shared",
+    "ConnectionString": "**ConnectionString**",
     "LogGeneratedSql": "false",
     "MemoryCacheEnabled": "true",
     "MemoryCacheSliding": "true",
     "MemoryCacheMinDuration": "5",
     "DefaultItemsPerPage": "20"
   },
   "ElasticSearchSettings": {
```

Comment:
We need to add an issue to remove the ElasticSearchSettings.  There is a really old branch that utilizes Elastic ... but it is way out of date and it was basically a spike.